### DISCUSSION TITLE
引用assets文件夹内图片,github actions构建时报错

### DISCUSSION THREAD

**Discussion by jiange1236** [vuepress/core#889] [Q&A] [answered]
引用方式为typora默认的新建assets子文件夹的方式,上传至github部署时报错,求大神指导一下
报错日志如下:
[vite:asset] Could not load /home/runner/work/jiange1236.GitHub.io/jiange1236.GitHub.io/notes/work/设计.assets/Q0%7DK_%7D30WUFM.png (imported by notes/.vuepress/.temp/pages/work/设计.html.vue): ENOENT: no such file or directory, open '/home/runner/work/jiange1236.GitHub.io/jiange1236.GitHub.io/notes/work/设计.assets/Q0%7DK_%7D30WUFM.png'
✖ Compiling with vite - failed
详细报错信息可查看:
https://github.com/jiange1236/jiange1236.GitHub.io/runs/6422944653?check_suite_focus=true

**Comment by jiange1236** [comment, marked as answer]
thanks to @Mister-Hope ,
i delete the special characters in filename and it works!

**Reply by Mister-Hope** [reply]
I open an issue https://github.com/vuepress/vuepress-next/issues/890, you can check that.
